workers/exclusive-content/README.md

```diff
@@ -0,0 +1,92 @@
+# Exclusive Content Worker
+
+This cloudflare worker will encrypt any plain text given a payment pointer. Furthermore, it will decrypt the resulting cypher text if it can verify payment by a receipt verifier.
+
+## Testing
+
+Configure your `wrangler.toml` file.
+
+```sh
+npm install
+wrangler secret put MASTERKEY
+# enter a key to be stored with cloudflare worker
+wrangler preview --watch
+```
+
+## Deploying
+
+Configure your `wrangler.toml` file.
+
+```sh
+npm install
+wrangler secret put MASTERKEY
+# enter a key to be stored with cloudflare worker
+wrangler publish
+```
+
+## Endpoints
+
+### `POST /encrypt`
+
+Encrypts plaintext
+
+#### Request Body
+
+JSON object containing
+| Name | Type | Description | Example |
+| -- | -- | -- | -- |
+| pp | String | (proxy) payment pointer | "\$spsp.example.com/alice" |
+| pt | String | plain text | "Hello World" |
```

Comment:
we should determine the largest payload we can encrypt/decrypt in the time limit that workers allows. If we find it to be restrictive at all I think we should consider the approach of returning keys instead of passing plaintext/ciphertext to the server directly

Reply:
I think we have decided to move encryption and decryption outside of the worker. That way, the original exclusive content never leaves the content creator's browser.